src/components/UserInfo.js

```diff
@@ -11,9 +12,19 @@ class UserInfo {
     };
   }
 
-  setUserInfo({ name, description }) {
+  setUserInfo({ name, about }) {
     this._name.textContent = name;
-    this._description.textContent = description;
+    this._description.textContent = about;
+  }
+
+  // getAvatar() {
+  //   return {
+  //     avatar: this._avatar.value,
+  //   };
+  // }
+
+  setAvatar({ avatar }) {
```

Comment:
It's great to have a setter for the avatar as it's controlled by a different form